engine/src/main/java/org/destinationsol/game/Console.java

```diff
@@ -208,7 +208,7 @@ public static Console getInstance() {
      */
     public void registerCharEntered(char c) {
         if (isActive) {
-            if (c == '\r') {
+            if (c == '\r' || c == '\n') {
```

Comment:
How did this even work beforehand? (Don't ask me, I'm the original author)

Reply:
I don't know... Maybe the merge of lwjgl3 changed the behaviour?